src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx

```diff
@@ -29,35 +36,41 @@ export const WalletMoreInfoCategory = ({
   const { t } = useTranslation("page-wallets-find-wallet")
 
   return (
-    <Box width="full" mt={4} _first={{ mt: 2 }}>
-      <Heading
-        as="h4"
-        lineHeight={1.4}
-        fontSize="md"
-        fontWeight={500}
-        mx="0.2rem"
-        mb={2}
-      >
+    <Box mx={{ base: 0, md: 0, lg: 2 }}>
+      {/* Category label */}
+      <Heading as="h4" lineHeight={1.4} fontSize="md" fontWeight="bold" mb={2}>
         {headingLabel}
       </Heading>
-      <Stack gap={2} wrap="wrap">
+
+      {/* Supported features */}
+      <UnorderedList m={0}>
         {orderedFeatureDropdownItems.map((feature) => {
           const featureColor = wallet[feature.filterKey!] ? "text" : "secondary"
 
+          const FeatureIcon = () => (
+            <Icon
+              as={
+                wallet[feature.filterKey!]
+                  ? GreenCheckProductGlyphIcon
+                  : WarningProductGlyphIcon
+              }
+              fontSize="1rem"
```

Comment:
```suggestion
              fontSize="md"
```

or, we may not even need this since I believe that would be the default size.